Makefile

```diff
@@ -493,11 +493,12 @@ test-e2e:
 	$(GO) run ./cmd/scylla-operator-tests run $(GO_TEST_E2E_EXTRA_ARGS)
 .PHONY: test-e2e
 
-test-version-script:
+test-scripts:
 	./hack/lib/tag-from-gh-ref.sh
-.PHONY: test-version-script
+	./hack/test/gke/xfs-formatter/tune2fs_test.sh
```

Comment:
nit: maybe less dirs :)

Reply:
What would you change it to? I thought it would be nicer if the test file structure reflected `hack`.